### DISCUSSION TITLE
GitHub page doesn't update

### DISCUSSION THREAD

**Discussion by Mlocik97** [community/community#22326] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I have problem with GitHub Pages in my private repo. I published page, it worked OK few days, but for last 2 days I did some updates, but I see only version, which is 2 days ago. I thought it can be because of syntax error.  I found this thread: <a href="https://stackoverflow.com/questions/24713112/my-github-page-wont-update-its-content" rel="nofollow noopener">https://stackoverflow.com/questions/24713112/my-github-page-wont-update-its-content</a> about it…</p>
<p>Because it isn’t updating from time, when I added this code: </p>
<pre><code>&lt;div data-ng-bind-html="::something"&gt;&lt;/div&gt;
</code></pre>
<p>I removed it, but problem still. </p>
<p>sry for my bad English.</p>
<p>E: ps. if I copy sources to another repo, and here I publish gh page, it’s working,… but i’m not owner of original repo. And I want it to work in original repo.</p>

**Comment by sis-c** [comment, marked as answer]
<p>It seems that about 12 hours or so ago a change was rolled out that does not allow Pages to be published from private repos on free accounts - or student accounts either :frowning:</p>

**Comment by Mlocik97** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/14b3a43d459f9a3f9bd175d19cb7c3551d6422d2.png" data-download-href="/uploads/short-url/2X8qDFQlXx8Bkc2w1ZZ5Tcc871U.png?dl=1" title="Bez názvu.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/34808350/181070191-e022ab47-ff77-4505-b591-918f8d99172d.png" alt="Bez názvu.png" data-base62-sha1="2X8qDFQlXx8Bkc2w1ZZ5Tcc871U" width="690" height="96" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/14b3a43d459f9a3f9bd175d19cb7c3551d6422d2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Bez názvu.png</span><span class="informations">761×106 5.08 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>aby why page is not free, when repo is free? there nothing is needed from server, difference for GitHub is ± 0.000001% … they can allow pages also for free as well. And while it can be already bypassed, it’s rly useless to let this in state, like it’s now. You can make repo public and then immediately private, and site will be builded.</p>

**Comment by blainerobertson** [comment]
<p>To be clear then, repos must be public and not have been granted educational status? Then the repo will refresh and distribute updates correctly. Is this correct?</p>

**Comment by nadiajoyce** [comment]
<p>Yes, as long as your trying to publish your Pages site from a public repo, it should work. If you’re experiencing issues with this, I recommend reaching out to <a href="https://github.com/contact" rel="nofollow noopener">private support</a> so they can take a closer look at your specific account.</p>
